packages/roosterjs-editor-dom/lib/utils/tableInfo.ts

```diff
@@ -0,0 +1,42 @@
+import { TableFormat } from 'roosterjs-editor-types';
+
+const TABLE_STYLE_INFO = 'roosterTableInfo';
+
+/**
+ * Save the format info of a table
+ * @param table The table the info will be saved
+ * @param format The format of the table
+ */
+export function saveTableInfo(table: HTMLTableElement, format: Partial<TableFormat>) {
+    if (table) {
+        table.dataset[TABLE_STYLE_INFO] = JSON.stringify(format);
+    }
+}
+

```

Comment:
I think you could call this in the applyFormat in VTable, so there is no need to call separately it every time when changing the table style.

Reply:
Also check if the format param have value.
Otherwise a undefined would be added to the dataset